mirrord/operator/src/client.rs

```diff
@@ -69,7 +71,17 @@ impl OperatorApi {
         let target_api: Api<TargetCrd> =
             get_k8s_resource_api(&client, target_config.namespace.as_deref());
 
+        let session_id = std::env::var(MIRRORD_OPERATOR_LAYER_SESSION)
```

Comment:
Coud you add docs to the public `discover` method above, saying that it now uses env? It's always surprising, when function's behavior depends on something more than its arguments